examples/echo-server/Cargo.toml

```diff
@@ -5,5 +5,5 @@ edition = "2021"
 
 [dependencies]
 http = "1"
-tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```

Comment:
This will be reverted before the PR is merged. I was a bit too lazy to write tests this morning and there is nothing wrong with that. I'm trying to get as much done as I can before I either find a job or go bankrupt. In either case, I'll have less time to work on this project. In efforts to get as close to an official, stable 2.0 release in the required time frame, I'm stealing the time that I had budgeted for writing tests under the assumption that once it's confirmed to work, I can probably get some tests implemented quickly with something generated from ai as a starting point.